packages/tokamak/contracts-bedrock/scripts/verification/UpdateVerificationValues.s.sol

```diff
@@ -0,0 +1,126 @@
+    _multisigWallet = vm.envAddress('MULTISIG_WALLET');
+
+    tokamakDAO = vm.envAddress('TEST_TOKAMAK_DAO_ADDRESS');
+    foundation = vm.envAddress('TEST_FOUNDATION_ADDRESS');
+    safeThreshold = 3;
+    safeWalletProxy = vm.envAddress('SAFE_WALLET');
+    safeWalletImplementation = IGnosisSafe(safeWalletProxy).masterCopy();
+    safeOwnersCount = 3;
+
+    systemConfigProxyAddress = vm.envAddress('TEST_SYSTEM_CONFIG_PROXY');
+    proxyAdminAddress = vm.envAddress('TEST_PROXY_ADMIN_ADDRESS');
+  }
+
+  function run() public {
+    setUp();
+
+    L1ContractVerification l1Verification = L1ContractVerification(
+      _proxyAddress
+    );
+    IMultiSigWallet multiSig = IMultiSigWallet(_multisigWallet);
+
+    console.log('Preparing to call setSafeConfig via multisig...');
+    bytes memory setSafeConfigData = abi.encodeWithSelector(
+      L1ContractVerification.setSafeConfig.selector,
+      tokamakDAO,
+      foundation,
+      3,
+      safeWalletImplementation.codehash,
+      safeWalletProxy.codehash,
+      3
+    );
+
+    vm.broadcast(_multisigOwner1);
+    multiSig.submitTransaction(_proxyAddress, 0, setSafeConfigData);
+    uint txIndexSetSafeConfig = multiSig.getTransactionCount() - 1;
+    console.log(
+      'setSafeConfig transaction submitted by owner 1, txIndex:',
+      txIndexSetSafeConfig
+    );
+
+    vm.broadcast(_multisigOwner2);
+    multiSig.confirmTransaction(txIndexSetSafeConfig);
+    console.log('setSafeConfig transaction confirmed by owner 2');
+
+    vm.broadcast(_multisigOwner3);
+    multiSig.confirmTransaction(txIndexSetSafeConfig);
+    console.log('setSafeConfig transaction confirmed by owner 3');
+
+    vm.broadcast(_multisigOwner1);
+    multiSig.executeTransaction(txIndexSetSafeConfig);
+    console.log('setSafeConfig transaction executed by owner 1');
+
+    console.log('Preparing to call setLogicContractInfo via multisig...');
+    bytes memory setLogicContractInfoData = abi.encodeWithSelector(
+      L1ContractVerification.setLogicContractInfo.selector,
+      systemConfigProxyAddress,
+      proxyAdminAddress
+    );
+
+    vm.broadcast(_multisigOwner1);
+    multiSig.submitTransaction(_proxyAddress, 0, setLogicContractInfoData);
+    uint txIndexSetLogic = multiSig.getTransactionCount() - 1;
+    console.log(
+      'setLogicContractInfo transaction submitted by owner 1, txIndex:',
+      txIndexSetLogic
+    );
+
+    vm.broadcast(_multisigOwner2);
+    multiSig.confirmTransaction(txIndexSetLogic);
+    console.log('setLogicContractInfo transaction confirmed by owner 2');
+
+    vm.broadcast(_multisigOwner3);
+    multiSig.confirmTransaction(txIndexSetLogic);
+    console.log('setLogicContractInfo transaction confirmed by owner 3');
+
+    vm.broadcast(_multisigOwner1);
+    multiSig.executeTransaction(txIndexSetLogic);
+    console.log('setLogicContractInfo transaction executed by owner 1');
+
+    console.log('Multisig call tests completed.');
```

Comment:
If it is a script that will actually be used when an update is needed, I don't think the word test is appropriate.

suggestion: "Verification factor update completed"

Reply:
It seems your update is not pushed yet. Please check it.